podpac/core/interpolation/interpolator.py

```diff
@@ -198,6 +198,8 @@ class Interpolator(tl.HasTraits):
     methods_supported = tl.List(tl.Unicode())
     dims_supported = tl.List(tl.Unicode())
     spatial_tolerance = tl.Float(allow_none=True, default_value=np.inf)
+    time_tolerance = tl.Union([tl.Unicode(), tl.Instance(np.timedelta64, allow_none=True)])
```

Comment:
@mpu-creare is the `time_tolerance` and `alt_tolerance` okay here? Without these, some of the tests throw warnings (and will throw exceptions in traitlets 5). There is another way I can fix it, though, if they really shouldn't be here in the base class.

Reply:
These should not be in the base class -- not all the interpolators support those parameters... in fact spatial tolerance shouldn't be there either. Otherwise these parameters might be silently ignored, #340 . 

So yes, right now the way parameters for interpolators are being set will break in traitlets 5, I didn't consider that. If you can fix it, that would be great... perhaps we should do something similar as we did with the nodes and tag the interpolation parameters... feel free to push this fix off to another PR.

Reply:
I think we need to fix it now. The problem is the super call in the mase Interpolator `__init__`. It's easy to just extract the tolerance arguments in the child class `__init__` methods that need them. But, `spatial_tolerance` and `alt_tolerance` are both used in the base Interpolator `_loop_helper`, so it's kind of confusing or even buggy, separate from the traitlets issue. They are also all documented in the COMMON_INTERPOLATOR_DOCS, though that is less important.

Reply:
I greatly simplified `_looper_helper`, which was a holdover from Phase I. Most of that functionality now lives in `interpolation_manager`.